client/src/components/AddVideo.js

```diff
@@ -9,41 +8,39 @@ const AddVideo = ({ videoData, setVideoData }) => {
   const handleAddVideo = (e) => {
     e.preventDefault();
     const newVideoData = {
+      id: Math.floor(Math.random() * 10000),
       title: title,
       url: url,
+      rating: 0,
     };
     setVideoData(videoData.concat(newVideoData));
   };
 
   return (
     <section className="addVideoContainer">
-      <h4 onClick={() => setAddVideo(!addVideo)}>Add Video</h4>
+      <button className="btn addVidBtn " onClick={() => setAddVideo(!addVideo)}>
+        <h5>Add Video</h5>
+      </button>
       {addVideo && (
         <form>
           <div>
-            <label>
-              Title
-              <input
-                className="input"
-                name="title"
-                type="text"
-                required
-                placeholder="title"
-                onChange={(e) => setTitle(e.target.value)}
-              ></input>
-            </label>
```

Comment:
I would always try to have a label for inputs if possible (helps massively with accessibility) if not, you could add an aria-label to the input for example `aria-label="title"`